ci-operator/step-registry/azure/cluster/public-to-private/azure-cluster-public-to-private-commands.sh

```diff
@@ -0,0 +1,204 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# save the exit code for junit xml file generated in step gather-must-gather
+# pre configuration steps before running installation, exit code 100 if failed,
+# save to install-pre-config-status.txt
+# post check steps after cluster installation, exit code 101 if failed,
+# save to install-post-check-status.txt
+EXIT_CODE=101
+trap 'if [[ "$?" == 0 ]]; then EXIT_CODE=0; fi; echo "${EXIT_CODE}" > "${SHARED_DIR}/install-pre-config-status.txt"' EXIT TERM
+
+# az should already be there
+command -v az
+az --version
+
```

Comment:
@jinyunma if the current step only works for 4.17+, how about adding an OCP version check here, if it was added to 4.16/4.15 jobs by accident, it can quit directly.

Reply:
sure, updated.